lib/assisted_workflow/addons/pivotal.rb

```diff
@@ -1,18 +1,30 @@
 require "assisted_workflow/exceptions"
 require "assisted_workflow/addons/base"
-require 'pivotal_tracker'
+require "tracker_api"
 
 # wrapper class to pivotal api client
 module AssistedWorkflow::Addons
+
+  class PivotalStory < SimpleDelegator
+    def initialize(story)
+      super
+    end
+
+    def owners_str
+      url = "/projects/#{project_id}/stories/#{id}/owners"
+      client.get(url).body.map{|owner| owner["name"]}.join(", ")
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.